scout/adapter/mongo/filter.py

```diff
@@ -138,7 +138,7 @@ def audit_filter(self, filter_id, institute_obj, case_obj, user_obj, category="s
             target = VARIANTS_TARGET_FROM_CATEGORY.get(category)
 
             case_name = case_obj.get("display_name")
-            link = url_for(target, case_name=case_name, institute_id=institute_obj.get("_id"))
+            link = url_for(target, case_name=case_name, **filter_obj)
```

Comment:
I need to change this and collect the info from the variants page I'm on instead!